lib/jekyll/converters/scss.rb

```diff
@@ -179,18 +186,53 @@ def sass_configs
         )
       end
 
+      def sass_embedded_config(data)
+        {
+          :data                => data,
+          :file                => file_path,
+          :indented_syntax     => syntax == :sass,
+          :include_paths       => sass_load_paths,
+          :output_style        => sass_style,
+          :source_map          => sourcemap_required?,
+          :out_file            => output_path,
+          :omit_source_map_url => !sourcemap_required?,
+          :source_map_contents => true,
+        }
+      end
+
       def convert(content)
+        case sass_implementation
+        when "sass-embedded"
+          Jekyll::External.require_with_graceful_fail("sass")
```

Comment:
The problem with this is that `Jekyll::External.require_with_graceful_fail` assumes that the gem is named after the entry-path (Ruby convention).
But with `sass-embedded`, that is not the case..
We need a new helper..

On that note, it would be better if `sass-embedded` exported this explicitly as part of the specification metadata:

```ruby
spec.metadata["require_key"] = "sass" # any intuitive keyname will do
```
so that if in the future, you decide to rename the entry path as "sass/embedded" or "sass-embedded", downstream consumers won't be affected if they were to require via the metadata.

Reply:
@ashmaroli I just added a `lib/sass-embedded.rb` in `sass-embedded` gem and changed it to require that. Althoug this is against the [recommendation by rubygems](https://guides.rubygems.org/name-your-gem/), this seems to be fairly commonly in practice, and it solves the problem here.

Reply:
> Although this is against the [recommendation by rubygems](https://guides.rubygems.org/name-your-gem/), this seems to be fairly common in practice..

Yes, in fact all of the official jekyll plugins (the ones under jekyll org) follow the *illegal* convention.